examples/kubernetes/ceph-osd-v1-ds.yaml

```diff
@@ -17,9 +17,12 @@ spec:
       nodeSelector:
         node-type: storage
       volumes:
-        - name: devices
-          hostPath:
-            path: /dev
+# Comment back in to use direct device access.
+# Note that this will also require enabling privileged access and
```

Comment:
👎 
This is only true when you do journal collocation and when you use a journal file for Ceph.
If you use a raw partition for the journal you will need access to that device.